### ISSUE TITLE
Problem with the branches 

### ISSUE THREAD

**Issue by Azbo400** [xGabey/Garners-Adventure#9]
Hey guys! There is a problem with the repo and we need to fix it so that we can make changes to the master branch. 

What happened is that the `canvas` branch is the same things as `master` however it is 7 commits behind the files in master, and the files that are in `master` are the ones that should be in `xGabey-canvas` and vice-versa so xGabey-canvas contains what should be in the master branch.

In other words, we need to switch the branch `xGabey-canvas` with `master` and get rid of `canvas`

**Comment by xGabey** [comment]
Hey, that's exactly what I was trying to do yesterday. I've looking it up, but I haven't found anything yet.

**Comment by Azbo400** [comment]
In addition to that when I push my design changes to master, masters files will get messed up and I am not ready to push my files so I cant pull them in and push it and then put the canvas files in a folder. Can somebody else do that.

**Comment by Azbo400** [comment]
Ah, I have an idea go onto the gitter chat!

**Comment by Azbo400** [comment]
👍 all fixed
